spec/sidekiq/jobs/inning_rollover_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.inline!
+
+RSpec.describe InningRolloverJob, type: :job do
+  before :each do
+    Sidekiq::Testing.fake!
+    @inning1 = create(:inning, :is_current)
+    @inning2 = create(:inning, :not_current_future)
+  end
```

Comment:
In the [Sidekiq Testing Docs](https://github.com/sidekiq/sidekiq/wiki/Testing) it says that requiring `sidekiq/testing` automatically sets the testing mode to `fake`, so I don't think you need to explicitly set it to `fake` in your before each.